serverless/serverless-getting-started.adoc

```diff
@@ -13,23 +13,27 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 
 [id="how-it-works_{context}"]
 == How {ServerlessProductName} works
-Developers on {ServerlessProductName} can use the provided Kubernetes-native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
-For information about installing {ServerlessProductName}, see xref:../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
 
-{ServerlessProductName} on {product-title} enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
+Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
+
+{ServerlessProductName} on {product-title} enables stateless, serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```

Comment:
Seems like we should lose the comma after 'stateless' now

Reply:
Kinda wondered this myself too, think you're right